### PR TITLE
New command schedule

### PR DESCRIPTION
**Pull request by RishiVaya**
Created endpoint to create new scheduled command.

### CODE REVIEW

File: src/leo-server-app/src/routes/schedule.ts [hunk 1]
```diff
@@ -138,6 +148,58 @@ const checkSatellitePermissionList = async (
...
+      query.command
+    );
+
+    if (!isCommandInSatelliteCriteria) {
+      return res.status(500).json({ error: "Invalid command sequence" });
+    }
+
+    // TODO:  Check if command exists in the user's permission list for satellite unless they are admin
+
+    // add command record
```
Comment (quinnha) [inline src/leo-server-app/src/routes/schedule.ts:187]: Nit: "Add command record"

**Review by quinnha — COMMENTED** [review]
lgtm, why is this change needed?

**Comment by quinnha** [comment]
also im guessing tests are to be added later?

**Review by Umang-Rajkarnikar — APPROVED** [review]
lgtm

**Pull request merged**